src/main/java/au/org/ala/biocache/dto/DownloadRequestParams.java

```diff
@@ -354,4 +372,21 @@ public void setHubName(String hubName) {
     public void setDoiMetadata(Map<String, String> doiMetadata) {
         this.doiMetadata = doiMetadata;
     }
+
+    public void setEmailTemplate(String emailTemplate) {
+        if( validTemplates.contains(emailTemplate) )
```

Comment:
Use braces for the if condition to match the style for the rest of biocache-service